### PR TITLE
add missing props to CONTRIBUTORS.md

### PR DESCRIPTION
**Pull request by jeffpaul**
**Description**

Note that some of the references below are not the only contributions (commits, PR review, helpful issue input, etc.) from these folks, but merely the first ones I encountered while reviewing items committed to `trunk` since the initial branch commit.

@westonruter via  #51
@ntwb via #73
@juricav via #113
@sandstromer via #69
@jasmussen via #74
@melchoyce via #16
@riyadh1734 via #182
@desrosj via #223
@beafialho via #172
@clucasrowlands via #171
@otto42 via #28
@luminuu via #107
@felixarntz via #240

I still need to add the WordPress.org Usernames, but wanted to get this draft PR up now while I work to find those second pieces of info.

Relates to #59.

**Screenshots**

n/a

**Testing Instructions** 

n/a

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -5,32 +5,45 @@
...
 | @mtoensing | @mtoensing |
 | @nickcernis | @nickcernis |
 | @nielslange | @nielslange |
+| @ntwb |  |
+| @otto42 |  |
 | @pattonwebz | @williampatton |
 | @pbking | @pbking |
 | @ribaricplusplus | @ribaricplusplus |
 | @richtabor | @onemaggie |
+| @riyadh1734 |  |
```
Comment (Riyadh1734) [inline CONTRIBUTORS.md]: WP username | saju4wordpress

**Review by Riyadh1734 — APPROVED** [review]
Add WordPress Username

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -5,32 +5,45 @@
...
 | @mburridge | @mburridge |
 | @mtoensing | @mtoensing |
 | @nickcernis | @nickcernis |
 | @nielslange | @nielslange |
+| @ntwb | @netweb |
+| @otto42 | @otto42 |
 | @pattonwebz | @williampatton |
 | @pbking | @pbking |
 | @ribaricplusplus | @ribaricplusplus |
 | @richtabor | @onemaggie |
```
Comment (MaggieCabrera) [inline CONTRIBUTORS.md]: would you mind changing this to use Rich's wp.org username? looks like a copy-paste error here :D

**Review by kjellr — APPROVED** [review]
Thanks for pulling this together, @jeffpaul!

**Pull request merged**